docker-compose.yaml

```diff
@@ -31,6 +31,31 @@ services:
     ports:
       - "8081:8081"
 
+  minio:
```

Comment:
How do we ensure that this container is not part of the production build? Does it belong in a separate `docker-compose.ci.yaml`?

Reply:
I believe we are not using docker-compose for prod, just the image loaded in via kubernetes

Reply:
> I believe we are not using docker-compose for prod, just the image loaded in via kubernetes

Exactly. docker compose is not used for either image building or production runs so there's no worry in it being here. It's used for development only, and minio is also there for development and testing only, so belongs in the `docker-compose.yaml` (or maybe `docker-compose.override.yaml` if we were trying to do something only in dev and not in tests)

Reply:
> (or maybe docker-compose.override.yaml if we were trying to do something only in dev and not in tests)

We aren't using docker-compose for the tests either at the moment, the integration tests run with mocked external services using supertest